.github/workflows/publish-docker-image-ghcr.yml

```diff
@@ -41,12 +41,12 @@ jobs:
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: build and publish container
+      - name: build and publish image
         uses: docker/build-push-action@v6.7.0
         with:
           push: true
           file: Dockerfile-ghcr
-          platforms: linux/amd64,linux/arm64,linux/ppc64le
+          platforms: linux/amd64,linux/arm64,linux/ppc64le,windows/amd64
```

Comment:
does it actually work? As far as I can tell the base image used in https://github.com/signalfx/splunk-otel-java/blob/82043a5e67c54f7d7fa6d688c22a615fe6668c92/Dockerfile-ghcr#L7 does not support windows.

Reply:
@atoulme need your opinion here thanks.